money-monkey-web/app/lib/db/user.ts

```
@@ -0,0 +1,19 @@
+
+export async function createUser(userId: string): Promise<void> {
+  const existing = (
+    await db
+      .select({ name: categoryTable.name })
+      .from(categoryTable)
+      .where(eq(categoryTable.user_id, userId))
+  ).map((e) => e.name);
+
+  var categories = DEFAULT_CATEGORIES.filter(
+    (cat) => !existing.includes(cat.name)
+  ).map((cat) => ({ ...cat, user_id: userId }));
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)